src/routes.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { createBrowserRouter } from "react-router-dom";
+
+import { PageLayout } from "./layouts";
+import { EnvironmentDetails } from "./features/environmentDetails";
+import { EnvironmentCreate } from "./features/environmentCreate";
+
+/**
+ * Define URL routes for the single page app
+ */
+export const router = createBrowserRouter([
```

Comment:
On review of this, I'm thinking that we actually shouldn't use one of the data API routers - see [Picking a Router](https://reactrouter.com/en/main/routers/picking-a-router). Not if we keep using RTK. 

That said, I'm also thinking we should move away from component-based data fetching. Some discussions about why:

- [Component fetching and render fetch chains](https://remix.run/blog/remixing-react-router#component-fetching-and-render-fetch-chains)
- [Can I use React without a framework?](https://react.dev/learn/start-a-new-react-project#can-i-use-react-without-a-framework)

So left it as a router with data API because I guess I could see us swinging either way down the road.

Reply:
Started a discussion:

- https://github.com/conda-incubator/conda-store-ui/discussions/396